### DISCUSSION TITLE
Blockchain exception after docker image update

### DISCUSSION THREAD

**Discussion by Isanderthul** [guydavis/machinaris#543] [Q&A] [answered]
**Describe the bug**
upgraded the docker image from 0.6.7:test to 0.6.8:latest
after the upgrade, when viewing the blockchains Chia, the page is filled with exception logs

**Additional context & screenshots**
```
Exception from 'show' Traceback (most recent call last):
  File "/chia-blockchain/chia/cmds/show.py", line 41, in show_async
    blockchain_state = await client.get_blockchain_state()
  File "/chia-blockchain/chia/rpc/full_node_rpc_client.py", line 27, in get_blockchain_state
    response = await self.fetch("get_blockchain_state", {})
  File "/chia-blockchain/chia/rpc/rpc_client.py", line 49, in fetch
    raise ValueError(res_json)
ValueError: {'error': "b'\\xd7\\x80\\xd2,z\\x87\\xc9\\xe0\\x1d\\x98\\xb4\\x9a\\t\\x10\\xf6p\\x1c;\\x95\\x01WA1k?\\xda\\x04.]{\\x81\\xd2'", 'success': False}
```
<img width="1870" alt="Screenshot 2022-01-19 at 15 58 38" src="https://user-images.githubusercontent.com/3693197/150145162-9dc637d8-a9f0-4416-a98e-38b759dea6c9.png">

**Comment by Isanderthul** [comment]
so I thought that I would try out changing the docker image to 0.6.8:test, seeing how the previous was on test
now the blockchain error has gone away and been replaced with this:
 - oh this makes me feel 🤮 - 

<img width="1605" alt="Screenshot 2022-01-19 at 16 15 43" src="https://user-images.githubusercontent.com/3693197/150148241-72e34705-401c-4cb6-b15a-14749f00a28b.png">

**Comment by guydavis** [comment, marked as answer]
Sorry to hear that.  Not uncommon unfortunately for Chia's database to become corrupted.  Affects regular Chia users and also Machinaris (rests on top of Chia services). 

If you don't have a backup and don't want to sync from zero, perhaps you should [download a database](https://github.com/guydavis/machinaris/wiki/Troubleshooting#reset-chia-configwalletblockchain).  Minutes instead of days.

**Comment by Isanderthul** [comment]
ok I will start downloading, 
strangely the filesystem shows a 43GB db...
```
root@Gen8:/mnt/user/appdata/machinaris/mainnet/db# ls -alh
total 43G
drwxr-xr-x 1 root root  152 Jan 19 16:13 ./
drwxr-xr-x 1 root root   95 Nov  7 18:48 ../
-rw-r--r-- 1 root root  43G Jan 19 10:32 blockchain_v1_mainnet.sqlite
-rw-r--r-- 1 root root  32K Jan 19 16:17 blockchain_v1_mainnet.sqlite-shm
-rw-r--r-- 1 root root    0 Jan 19 16:13 blockchain_v1_mainnet.sqlite-wal
-rw-r--r-- 1 root root 5.1M Jan 19 16:02 peer_table_node.sqlite
```

**Comment by Isanderthul** [comment]
after 30 minutes, the error has come back. I am going to get a new db image, and consider this current one corrupted 😢 
<img width="1802" alt="Screenshot 2022-01-19 at 17 06 08" src="https://user-images.githubusercontent.com/3693197/150157986-ec594bad-77b8-4c7e-a5b8-fd15bf557063.png">
